evalml/tests/automl_tests/test_automl.py

```diff
@@ -407,6 +408,16 @@ def test_automl_bad_data_check_parameter_type():
         automl.search(data_checks=[MockDataCheckErrorAndWarning])
 
 
+def test_validate_data_check_n_splits(X_y_multi):
+    X, y = X_y_multi
+    data_split = make_data_splitter(X, y, problem_type='multiclass', n_splits=4, random_state=42)
+
+    automl = AutoMLSearch(X, y, problem_type="multiclass", max_iterations=1, n_jobs=1, data_splitter=data_split)
+    automl.search(data_checks='auto')
+
+    assert automl.data_splitter.get_n_splits() == 4
```

Comment:
Hm, I think what we want to check here is actually that the data check got n_splits = 4 to confirm that the correct number of CV folds was passed to the data check.

Reply:
Thanks for catching that!